### PR TITLE
TDP-4110 GET /calls

### PR DESCRIPTION
**Pull request by scapps-bw**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

## Brief Summary of changes

Adding new GET /calls search endpoint. These docs changes add documentation for the endpoint and clarify in certain places that the other call data endpoint uses call ID rather than a search.

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4110.s3-website-us-east-1.amazonaws.com/index.html

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4110.s3-website-us-east-1.amazonaws.com/index.html

File: voice/methods/calls/getCalls.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+//COMING SOON
+```
+
+### Example 2 of 2: Retrieve calls by query
+
+{% sample lang="http" %}
+
+```bash
+curl -X GET \
+    --url "https://voice.bandwidth.com/api/v2/accounts/{accountId}/calls?pageSize=10&minStartTime=2020-07-13T12:00:000Z&maxStartTime=2020-07-13T13:00:000Z&to=+15552311778&from=+15552311772&state=disconnected&disconnectCause=hangup&disconnectCause=timeout" \
```
Comment (n-miles) [inline voice/methods/calls/getCalls.md]: The query parameters need to be escaped

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4110.s3-website-us-east-1.amazonaws.com/index.html

**Pull request closed without merging**